### PR TITLE
match master_sword.c

### PR DESCRIPTION
**Pull request by EpochFlame**

### CODE REVIEW

File: include/master_sword.h [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#ifndef GUARD_MASTERSWORD_H
+#define GUARD_MASTERSWORD_H
```
Comment (jiangzhengwenjz) [inline include/master_sword.h:2]:
```suggestion
#ifndef GUARD_MASTER_SWORD_H
#define GUARD_MASTER_SWORD_H
```

File: src/master_sword.c [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#include "master_sword.h"
+#include "kirby.h"
+#include "functions.h"
+
+static void sub_08025434(struct Object2* arg0);
+static void sub_08025440(struct Object2* arg0);
+
+struct Object2* CreateMasterSwordNonCollectible(struct Object* arg0, u8 arg1) {
```
Comment (jiangzhengwenjz) [inline src/master_sword.c:8]:
```suggestion
void* CreateMasterSwordNonCollectible(struct Object* arg0, u8 arg1) {
```
CreateXXX are placed in an array so we make all of them return void *.

**Review by jiangzhengwenjz — CHANGES_REQUESTED** [review]

File: src/master_sword.c (hunk 2 above)
Comment (EpochFlame) [reply src/master_sword.c:8]: will do, thanks for the feedback

**Comment by EpochFlame** [comment]
updated the create function accordingly

File: include/master_sword.h [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#ifndef GUARD_MASTER_SWORD_H
+#define GUARD_MASTER_SWORD_H
+
+#include "data.h"
+#include "global.h"
+#include "object.h"
+
+struct void* CreateMasterSwordNonCollectible(struct Object* arg0, u8 arg1);
```
Comment (jiangzhengwenjz) [inline include/master_sword.h:8]:
```suggestion
void* CreateMasterSwordNonCollectible(struct Object* arg0, u8 arg1);
```

**Review by jiangzhengwenjz — APPROVED** [review]

**Pull request merged**